packages/go/openapi/doc/openapi.json

```diff
@@ -14033,6 +14238,60 @@
           }
         }
       },
+      "model.auth-provider": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "integer",
+            "format": "int32",
+            "description": "Unique identifier for the SSO provider"
+          },
+          "name": {
+            "type": "string",
+            "description": "Name of the SSO provider"
+          },
+          "type": {
+            "type": "string",
+            "description": "Type of SSO provider (SAML or OIDC)"
+          },
+          "slug": {
+            "type": "string",
+            "description": "URL-friendly identifier for the provider"
+          },
+          "details": {
```

Comment:
I'm realizing now that we already have a `saml-provider` definition that I believe can be used for the details:
https://github.com/SpecterOps/BloodHound/blob/582ee9bbf1a77485e6e0fc3239fec877b08c0fb5/packages/go/openapi/src/schemas/model.saml-provider.yaml

Might warrant adding an `sso-provider` and `oidc-provider`, sorry I missed that on the first review!